source/dbo/Views/Power BI Reporting/vwMissingAntibioticMappings.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE VIEW [dbo].[vwMissingAntibioticMappings]
+AS
+
+SELECT su.AntibioticCode,
+	COUNT(DISTINCT a.ReferenceLaboratoryNumber) AS NumberOfSamples,
+	MIN(a.AuditCreate) AS DateFirstIntroduced,
+	MAX(a.AuditCreate) AS LastSeen
+FROM [$(Labbase2)].dbo.Susceptibility su
+	INNER JOIN [$(Labbase2)].dbo.Anonymised a ON a.LabDataID = su.LabDataID
+WHERE su.AntibioticCode NOT IN (SELECT AntibioticCode FROM [$(NTBS_Specimen_Matching)].dbo.AntibioticMapping)
+	-- Don't warn about not being able to map things that aren't antibiotics
+	AND su.AntibioticCode NOT IN ('New', 'Gynaecological', 'Other tissues')
```

Comment:
As agreed on the ticket we are only ignoring the `AntibioticCode`s that aren't actually antibiotics. The ones that we haven't mapped because we don't care about them and they haven't come up in a TB case yet are allowed to come through, in case they ever are seen on a case with `IsAtypicalOrganismRecord = 0`